bin/_mocha

```diff
@@ -497,13 +510,16 @@ if (!files.length) {
 }
 
 // resolve
-
+fileArgs = fileArgs.map(path => resolve(path));
 files = files.map(path => resolve(path));
 
 if (program.sort) {
   files.sort();
 }
 
+// add files given through --file to be ran first
+files = fileArgs.concat(files);
```

Comment:
okay, so `--sort` doesn't work with `--file`, which makes sense.  that should be noted somewhere if it isn't already.

Reply:
Added in `docs/index.md`